src/pdm/cli/utils.py

```diff
@@ -156,6 +157,36 @@ def parse_known_args(self, args: Any = None, namespace: Any = None) -> Any:
         return args, argv
 
 
+def find_similar_text(origin_name: str, target_names: list[str], ratio: float = 0.5) -> list[str]:
+    results = []
+    for name in target_names:
+        ratio = SequenceMatcher(None, origin_name, name).ratio()
+        if ratio > 0.4:
+            results.append((name, ratio))
+    return [name for name, _ in sorted(results, key=lambda x: x[1], reverse=True)]
+
+
+class ArgumentParserSimilarComandUtil:
+    color = {
+        "red": "\033[91m",
+        "green": "\033[92m",
+        "yellow": "\033[93m",
+    }
```

Comment:
You can use this utility to get colorized message with rich:
https://github.com/pdm-project/pdm/blob/1fd94bd5044ea10bbc814d92144a2d1a1bd663fd/src/pdm/termui.py#L58-L70

For example

```python
message = style("[success]This text is green[/]")
```

Reply:
nice ,I will dupdate it